polaris-react/.storybook/preview.js

```diff
@@ -79,10 +80,27 @@ export const globalTypes = {
   },
   ...gridOptions,
 };
+const views = {...breakpoints};
+delete views['breakpoints-xs'];
```

Comment:
Light suggestion to override this breakpoint value with `320px` (with a brief comment explaining why). That way we can still test `xs` (0-489px) viewports.

Reply:
I believe we do the same in with playroom cc @gwyneplaine